base/applications/fltmc/fltmc.cpp

```diff
@@ -220,7 +220,8 @@ PrintVolumeInfo(_In_ PVOID Buffer)
         VolName[FilterVolInfo->FilterVolumeNameLength] = UNICODE_NULL;
     }
 
-    (void)FilterGetDosName(VolName, DosName, 16);
```

Comment:
Interesting: Usually this `(void)` thing is precisely used to shut up compiler warnings about the non-checking of the returned result (see GCC, MSVC, where otherwise you would get the warning). But somehow this gets ignored?

Reply:
If you think your call succeeds anyway, then you either don't put the `warn_unused_result` or `_Must_inspect_result_` attribute in the first place, or you ASSERT on it in debug builds. I think that putting (void) is lazy and stupid, and it turns out that the GCC developers think like I do on this matter.